### DISCUSSION TITLE
lunar eclipse info

### DISCUSSION THREAD

**Discussion by ghost** [Stellarium/stellarium#3392] [Ideas]
So, in solar eclipses, there is eclipse info for it like magnitude, eclipse obscuration, width of umbra, etc. but there is no eclipse info that pops up in the lunar eclipse, so my suggestion is for info to show for the lunar eclipse when it's in progress. 
(This info appears in the left info tab that shows for info about objects)

**Comment by alex-w** [comment]
Magnitudes for lunar eclipses (umbral and penumbral) are already here.  The "eclipse obscuration" and "width of umbra" are senselessly for lunar eclipses. What's more?

**Reply by gzotti** [reply]
One could find shadow width at Lunar distance. But it's nothing usually given in almanachs either. Probably PA of shadow could be nice, But OP should specify his expectations and practical value. OK, OP has left already...
